webapp/channels/src/components/custom_status/custom_status_modal.tsx

```diff
@@ -206,7 +206,7 @@ const CustomStatusModal: React.FC<Props> = (props: Props) => {
         case FOUR_HOURS:
             return moment().add(4, 'hours').seconds(0).milliseconds(0).toISOString();
         case TODAY:
-            return moment().add(1, 'day').set({hour: 8, minute: 0}).toISOString();
+            return moment().endOf('day').add(1, 'minute').seconds(0).milliseconds(0).toISOString();
```

Comment:
The additional minute is not needed. See https://github.com/mattermost/mattermost-mobile/blob/main/app/components/custom_status/custom_status_expiry.tsx#L60

Reply:
I had to add the minute since end of day returns 23:59, which then sets the message as 'Until Today' which was also an issue as it didn't make sense, since the message should be 'Until Tomorrow'; see issue [#27324](https://github.com/mattermost/mattermost/issues/27324)

Reply:
If that is the case, them you'll need to change it also on mobile.

@asaadmahmood can you confirm what is the expected?

Thanks

Reply:
Cool, then it needs to be changed on mobile as well.

Thanks for the input

Reply:
Hey @enahum having issues getting android studio up and running due to memory issues, is it possible to run/test changes without android studio? If not, could a sister issue to this one be created so that changes can be made by someone who's already got everything set up?

Reply:
@Gesare5 Hi Ivy.  Thanks for your help on this.  I've created a new issue that is designated for mobile as to not cause delay in moving this forward.

Reply:
Sorry the internal ticket is: https://mattermost.atlassian.net/browse/MM-61845